core/modules/sink.h

```diff
@@ -5,6 +5,8 @@
 
 class Sink final : public Module {
  public:
+  Sink() { max_allowed_workers_ = Worker::kMaxWorkers; }
```

Comment:
I was not sure, I will look through the `Queue*` things to check, but at least `QueueInc` and `QueueOut` are not thread safe (they read from PMD queue).